### PR TITLE
Chat screen update

### PR DESCRIPTION
**Pull request by KiranChimple**
chat screen

### CODE REVIEW

File: app/components/ChatView.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+ import { Text,View,StyleSheet,Image} from 'react-native';
+import PropTypes from 'prop-types'
+import SvgUri from 'react-native-svg-uri'
+
+import { sendMessage, startChat } from '../redux/chat';
+
+ class ChatView extends Component{
+  constructor(props){
+    super(props)
+    this.state ={
```
Comment (talapadi) [inline app/components/ChatView.js]: Do we need this state?

File: app/components/ChatView.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+          <View style={styles.eachMsg}>
+        <Image source={{ uri: 'data:image/png;base64,' +this.props.user.image }}style={styles.userPic} />
+
+        {/* <Image source={{uri:'https://www.shareicon.net/data/2016/06/25/786533_people_512x512.png'}}style={styles.userPic} /> */}
+        <View style={styles.msgBlock}>
+          <Text style={styles.msgTxt}>{this.props.item.text}</Text>
+        </View>
+      </View>
+        );
+      }else if(this.props.item.sender !== this.props.user._id ){
```
Comment (talapadi) [inline app/components/ChatView.js]: Shouldn't this be just else

File: app/components/MessageList.js [hunk 3]
```diff
@@ -1,30 +1,44 @@
...
+ 
   _keyExtractor = (item, index) => item._id
 
-  _renderItem = ({item}) => (
-    <View>
-      <Text>
-        {item.text}
-      </Text>
+  _renderItem = ({item,user,friend}) => (
+    <View >
```
Comment (talapadi) [inline app/components/MessageList.js]: Do we need <View> outside?

**Review by talapadi — CHANGES_REQUESTED** [review]
Some comments are made inline. Also try to make the indentation proper

**Comment by KiranChimple** [comment]
I have changed the code,I will update it again.

File: app/components/ChatView.js (hunk 1 above)
Comment (KiranChimple) [reply app/components/ChatView.js]: when we need to input some message. We may need to change the state

File: app/components/ChatView.js (hunk 2 above)
Comment (KiranChimple) [reply app/components/ChatView.js]: yes,only else works fine.

File: app/components/MessageList.js (hunk 3 above)
Comment (KiranChimple) [reply app/components/MessageList.js]: we don't need this outside .It will when we just pass in the component

**Pull request merged**